packages/blocks/src/api/registration.js

```diff
@@ -126,6 +133,16 @@ import { store as blocksStore } from '../store';
  *                                              then no preview is shown.
  */
 
+/**
+ * A subset of `WPBlock` type. Contains basic block's information for display reasons.
+ *
+ * @typedef {Object} WPBlockDisplayInformation
+ *
+ * @property {string}      title       Human-readable block type label.
+ * @property {JSX.Element} icon        Block type icon.
```

Comment:
Block types acceppt `WPIcon` that is quite complex structure:
```suggestion
 * @property {WPIcon|JSX.Element} icon        Block type icon.
```